README.md

```diff
@@ -1,24 +1,52 @@
+
 # IPsec-Recipe
+
 The IPsec recipe is an application that enables strongSwan to use [Infrastructure Application Interface](https://ipdk.io/documentation/Interfaces/InfraApp/), specifically the P4Runtime and OpenConfig gRPCs provided by the [Networking-Recipe](https://github.com/ipdk-io/networking-recipe).
 
 The strongSwan plugin available in this repository implements a policy-based IPsec enablement using security policy database (SPD) and security association database (SAD). The P4Runtime client introduces a programmable IPsec flow using P4 language for the SPD and OpenConfig for SAD configurability.
 
 * IPsec recipe is validated only on Intel&reg; IPU E2100 target
 * Refer to https://ipdk.io/documentation/Recipes/InlineIPsec/
-* YANG model for IPsec SAD: https://github.com/ipdk-io/openconfig-public/blob/master/release/models/ipsec/openconfig-ipsec-offload.yang
+* YANG model for IPsec offload: https://github.com/ipdk-io/openconfig-public/blob/master/release/models/ipsec/openconfig-ipsec-offload.yang
 * Reference P4 program to enable IPsec on DPDK target: https://github.com/ipdk-io/networking-recipe/tree/main/p4src/Inline_IPsec
 
-strongSwan Integration with Inline Crypto Engine
-========================================================
+## strongSwan Integration with Inline Crypto Engine
+
 * The user should start and stop the strongSwan services as described in strongSwan manual.
-* The strongSwan configuration files must be configured by user as described in 
+* The strongSwan configuration files must be configured by user as described in
```

Comment:
```suggestion
* strongSwan must be configured by user as described in
```
- This avoids saying "the **configuration** files must be **configured**," which is both redundant and nonsensical. You don't configure the _files_; you configure strongSwan _using_ the files.
- Should _strongSwan_ be capitalized at the beginning of a sentence? I see instances in the text that suggest otherwise.